CHANGES.md

```diff
@@ -1,10 +1,15 @@
 # Changes
 
 ## Unreleased - 2020-xx-xx
-* Implement Logger middleware regex exclude pattern [#1723]
 
+* Implement Logger middleware regex exclude pattern [#1723]
 [#1723]: https://github.com/actix/actix-web/pull/1723
 
+### Fixed
+
+* awc now uses `Accept-Encoding: identity` instead of `Accept-Encoding: br` when no compression feature is enabled
+[#1737](https://github.com/actix/actix-web/pull/1737)
+
```

Comment:
this should be in `awc/CHANGEs.md`

Reply:
-> 06841e111c58a30537455fb2e0ba8e72df0ffcfe